DIGITfiles/SequencesFromBlast.py

```diff
@@ -181,32 +191,53 @@ def writeToBestQueriesFile(listQueries, flankseq):
                 "\n"
             )
             pickGenome(allele, bestBitScores)
-    return  # bestBitScores
+    return
 
 
 def getBestQuery(genome, query):
     '''
-    Sort hits to identify best hit for a query. Find the percent difference
-    between the best and second best hits.
+    Sort hits to identify best hit for a query. Find the percent difference between the best and
+    second best hits.
     '''
-    bestQuery = allQueries[genome][query][0]
-    secondBestQuery = None
-    for i in range(1, len(allQueries[genome][query])):
-        if allQueries[genome][query][i].bitScore > bestQuery.bitScore:
-            secondBestQuery = bestQuery
-            bestQuery = allQueries[genome][query][i]
-        elif allQueries[genome][query][i].bitScore < bestQuery.bitScore:
+    bestQuery = allQueries[genome][query][0]  # Query
+    secondBestQuery = None  # Query
+    for i in range(1, len(allQueries[genome][
```

Comment:
If you want to be pythonic, consider using this instead:
```for query in allQueries[genome][query][1:]:```

Reply:
How would I reference a specific Query object if I did that? allQueries has a structure of 
{ genome1 :
      { 
           query1 : [ hit1, hit2, hit3 ],
           query2 : [ hit1, hit2 ]
      }
  genome2:
      ....
      
      so I am going through the list at allQueries[genome][query], where each item in that list will have the same query ID but otherwise different information

Reply:
"for query in ..." returns 'query' as the specific Query object for that iteration, so you can reference query objects one at a time as you go through the list.